### ISSUE TITLE
Cancelling frame loading after timeout does not stop axe run.

### ISSUE THREAD

**Issue by jkodu** [dequelabs/axe-core#929]
# Title:
Cancelling frame loading after timeout does not stop axe run.

## Description
When running `axe` on an `iframe`, whilst the option to wait for frame loading is set to a negligible amount `frameWaitTime: 1`, this causes the below error:

### Tests in below files: 
```js
**/test/integration/full/frame-wait-time/frame-wait-time.html
**/test/integration/full/frame-wait-time/frame-wait-time.js
```

### Error:
actual - `Expect axe._selectorData to be set up`, but is unhandled.
expected - `Axe in frame timed out: #frame`.

## Analysis:
- Believe this is due to `axe` attempting to run even after frame load failure is triggered. 
- Ideally the execution should be stopped and the error should be passed back to the `run` callback.

**Comment by straker** [comment]
Closed via #1070
